tools/disasm/variables.txt

```diff
@@ -16985,13 +16985,13 @@
     0x80C23BD0:("En_Ending_Hero5_InitVars","UNK_TYPE1","",0x1),
     0x80C23BF0:("D_80C23BF0","UNK_TYPE1","",0x1),
     0x80C23C10:("jtbl_80C23C10","UNK_PTR","",0x4),
-    0x80C241E0:("En_Ending_Hero6_InitVars","UNK_TYPE1","",0x1),
-    0x80C24200:("D_80C24200","UNK_TYPE1","",0x1),
-    0x80C24224:("D_80C24224","UNK_TYPE1","",0x1),
-    0x80C24248:("D_80C24248","UNK_TYPE1","",0x1),
-    0x80C2426C:("D_80C2426C","UNK_TYPE1","",0x1),
-    0x80C24280:("D_80C24280","UNK_TYPE1","",0x1),
-    0x80C24294:("D_80C24294","UNK_TYPE1","",0x1),
+    0x80C241E0:("En_Ending_Hero6_InitVars","ActorInit","",0x20),
+    0x80C24200:("D_80C24200","FlexSkeletonHeader*","[9]",0x24),
+    0x80C24224:("D_80C24224","AnimationHeader*","[9]",0x24),
+    0x80C24248:("D_80C24248","s32","[9]",0x24),
+    0x80C2426C:("D_80C2426C","Gfx*","[5]",0x14),
+    0x80C24280:("D_80C24280","TexturePtr","[5]",0x14),
+    0x80C24294:("D_80C24294","TexturePtr","[3]",0xc),
```

Comment:
If these are all in the C file now, and never referred to in the asm again, it's okay to not rename/update them here.